lib/linguist/generated.rb

```diff
@@ -58,6 +58,7 @@ def generated?
       node_modules? ||
       go_vendor? ||
       npm_shrinkwrap? ||
+      yarn_lock? ||
```

Comment:
@pchaigno  sorry for lack of ruby experience, could you help me with this error? 🙇 
```
1) Error:
TestFileBlob#test_generated:
Errno::ENOENT: No such file or directory - /home/travis/build/github/linguist/samples/Gemfile.lock
    /home/travis/build/github/linguist/lib/linguist/file_blob.rb:33:in `read'
    /home/travis/build/github/linguist/lib/linguist/file_blob.rb:33:in `data'
    /home/travis/build/github/linguist/lib/linguist/blob_helper.rb:319:in `block in generated?'
```